src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedInstanceMethodSymbol.cs

```diff
@@ -88,5 +88,12 @@ internal sealed override bool HasAsyncMethodBuilderAttribute(out TypeSymbol buil
             builderArgument = null;
             return false;
         }
+
+        internal sealed override int? TryGetOverloadResolutionPriority()
+        {
+            return null;
+        }
+
+        internal sealed override bool CanHaveOverloadResolutionPriority => false;
```

Comment:
>CanHaveOverloadResolutionPriority [](http://example.com/codeflow?start=38&length=33)

`OverloadResolution.RemoveLowerPriorityMembers()` ignores overloads when the first one returns `false` from `CanHaveOverloadResolutionPriority`. And `SynthesizedInstanceMethodSymbol` has many implementations. Are we sure that all of these implementations are cases where all other overloads will return `false` from `CanHaveOverloadResolutionPriority`? 

For instance, `SynthesizedRecordCopyCtor` inherits from this type, but other constructors may have `[OverloadResolutionPriority]`.